### PR TITLE
:rocket: publishes new lectern version

### PR DESCRIPTION
**Pull request by hlminh2000**

### CODE REVIEW

File: index.yaml [hunk 1]
```diff
@@ -2,35 +2,23 @@ apiVersion: v1
 entries:
   ego:
   - apiVersion: v1
-    created: "2020-02-15T22:14:30.745294704-05:00"
-    dependencies:
-    - name: postgres
-      repository: https://overture-stack.github.io/charts-server
-      version: 0.2.0
```
Comment (rtisma) [inline index.yaml]: @hlminh2000 why did the dependencies disappear

Comment (hlminh2000) [reply index.yaml]: hmmm I'm not sure, how was are the dependencies specified?

Comment (hlminh2000) [reply index.yaml]: I didn't touch ego though...

Comment (rtisma) [reply index.yaml]: @henro001 @hlminh2000 i found out why it was deleted! so when i use `helm v2.14.3` to create the index, it deletes the dependencies (happened for score and song), however when i use `helm v3.1.0` the dependencies are maintained. Ensure you are using v3.1.0 to create the index.yaml

**Review by rtisma — APPROVED** [review]

**Comment by blabadi** [comment]
do we want to wrap this up ?

**Comment by hlminh2000** [comment]
@blabadi yes! sorry this slipped through as I had a bunch of PRs open. will update and merge

**Comment by hlminh2000** [comment]
actually, latest lectern is already in. Will just close

**Pull request closed without merging**